bundle/manifests/authorino.kuadrant.io_authconfigs.yaml

```diff
@@ -72,7 +72,7 @@ spec:
             type: object
           spec:
             description: Specifies the desired state of the AuthConfig resource, i.e.
-              the authencation/authorization scheme to be applied to protect the matching
+              the authentication/authorization scheme to be applied to protect the matching
```

Comment:
I think the failing CI is due to this should be updated at the CRD at **authorino** instead as a separate PR:
- https://github.com/Kuadrant/authorino/blob/b2bde91bfc64b1d140f740069ba36ce81a3f3349/api/v1beta2/auth_config_types.go#L109
- https://github.com/Kuadrant/authorino/blob/b2bde91bfc64b1d140f740069ba36ce81a3f3349/api/v1beta3/auth_config_types.go#L110

And the manifests can be updated using `make manifests generate` in **authorino**. 

**Authorino operator** can then update it's own bundle/manifest/chart using it's own make command without the CI failing one that has merged 🤔